### PR TITLE
adding glslCanvas template for easy edit/share of glsl shaders

### PR DESCRIPTION
**Pull request by patriciogonzalezvivo**
This should make easy for people to edit and share shaders using:
- https://thebookofshaders.com/edit.php
- https://github.com/patriciogonzalezvivo/glslViewer
- https://marketplace.visualstudio.com/items?itemName=circledev.glsl-canvas

Please change the readme as you please. I don't have a pristine grasp of what works and doesn't in the cryptosphere : )

### CODE REVIEW

**Comment by andrevenancio** [comment]
Hey @patriciogonzalezvivo this looks great! I only have a question. How will the resize work? at the moment, the platform expects that the html content will be able to resize to window.innerWidth and window.innerHeight. (and its quite easy to change the size as the user changes the browser size, or enters/exits fullscreen mode). Is it possible to add an example on how you can scale to fullscreen?

or you think that the canvas will always be 500x500 and then its the css that changes it to `canvas { width: 100%; height: 100% }` ?

File: templates/html-glslcanvas-template/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Your title here</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta property="og:image" content="thumbnail.jpg" />
+    <link type="text/css" rel="stylesheet" href="style.css" />
```
Comment (andrevenancio) [inline templates/html-glslcanvas-template/index.html]: I dont think there's a style.css in the template is there? Can we remove it or create it and move the CSS from the HTML into that style.css file

**Comment by patriciogonzalezvivo** [comment]
Good cach. I also notice the thumbnail now is a png

Let me know if you found another one.

Thanks again!

**Comment by patriciogonzalezvivo** [comment]
Oh sorry, miss the comment about the resizing. GlslCanvas handles resizing of the `canvas` element this way https://github.com/patriciogonzalezvivo/glslCanvas/blob/master/src/GlslCanvas.js#L463-L485

I was relying on CSS to keep it at 100%. I guess that's not the way to do it. right?

**Comment by andrevenancio** [comment]
> Oh sorry, miss the comment about the resizing. GlslCanvas handles resizing of the `canvas` element this way https://github.com/patriciogonzalezvivo/glslCanvas/blob/master/src/GlslCanvas.js#L463-L485
> 
> I was relying on CSS to keep it at 100%. I guess that's not the way to do it. right?

that's actually fine!

**Comment by pablovitamin** [comment]
Hello. I am trying to work with the glslCanvas template on hicetnunc but I think there is something wrong. In fullscreen the canvas is offset .5 to the top. I thought that the problem was of my code until I tried to mint as an OBJKT the template and the same issue occured. Any thoughts? thanks for everything.

**Comment by patriciogonzalezvivo** [comment]
Hi @pablovitamin! I have been improving this template and working on a script that automatically makes the .zip file and a GIF here https://github.com/patriciogonzalezvivo/henerator/ 
I fixed the fullscreen issues. Let me know it that helps you with your issue

**Comment by pablovitamin** [comment]
Thanks Patricio! Ill check it and let you know! thanks again!

**Comment by AlejoHarris** [comment]
Hello @patriciogonzalezvivo, tried your script but is throwing me this error:

```Traceback (most recent call last):
  File "C:\Users\Alejo\Desktop\HeNerator\henerator.py", line 2309, in <module>
    generate(file, args)
  File "C:\Users\Alejo\Desktop\HeNerator\henerator.py", line 2239, in generate
    shader_in_file = open(shader_filename+'.frag', "r")
FileNotFoundError: [Errno 2] No such file or directory: 'ejndjhwghg6pwxsu9vvskh27yhk011sz.frag'
```
Any idea of what could it be?

**Comment by patriciogonzalezvivo** [comment]
Hi Alejo. Sorry for the issue, haven't tried it on windows (just Linux and MacOS). Did you install GlslViewer? Seams is not generating the precompiled .frag

**Comment by patriciogonzalezvivo** [comment]
Also, I think opening an issue on HeNerator repository will be more appropriate, to research further. Do you mind?

**Comment by AlejoHarris** [comment]
I'll open the issue on your repository

**Pull request merged**